app/code/Magento/MediaGalleryApi/Model/Directory/Blacklist.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryApi\Model\Directory;
+
+/**
+ * Directories blacklisted for media gallery. This class should be used for DI configuration.
+ *
+ * Please use the interface in the code (for constructor injection) instead of this implementation.
+ *
+ * @api
+ */
+class Blacklist implements BlacklistInterface
```

Comment:
what's the sense to have both BlacklistInterface and Blacklist marked as API ?

Reply:
The class is used for DI configuration, the interface for functional purposes

Reply:
The Blacklist class removed in favour to configuration